.env.example

```diff
@@ -0,0 +1,8 @@
+JSON_EXPORT=traceability_export.json
+BASE_URL=https://base_url_to_documentation/or/base_path_to_documentation
+IP_ADDRESS=0.0.0.0
+CONFIG_FILE=config.yml
+PRODUCT=11111
+PACKAGE_TAG=latest
+DATABASE_URL=bolt://neo4j:password@neo4j_db:7687
+SECRET_KEY=ARANDOMSECRETKEY
```

Comment:
How do all of these environment variables become available in the Docker image/container?

Reply:
via `docker-compose.yml`
```env_file:
      - .env```
The ones needed in `Dockerfile` are listed here:
```build:
      context: .
      args:
        CONFIG_FILE: ${CONFIG_FILE}
        JSON_EXPORT: ${JSON_EXPORT}
```